### PR TITLE
Feature/speaker form fed

### PR DESCRIPTION
**Pull request by lindseybradford**

### CODE REVIEW

File: templates/_components/static/list-events.twig [hunk 1]
```diff
@@ -50,8 +52,8 @@
                         <div class="list-event__detail | paragraphs -xs | mb-2">
                             <p>
                                 Dr. Francisco will be the featured speaker at the
-                                annual men’s retreat on Sat March 16, for Our
-                                Shepherd Lutheran Church in Birmingham, MI.
+                                                                annual men’s retreat on Sat March 16, for Our
```
Comment (johnnymueller) [inline templates/_components/static/list-events.twig]: Is this spacing intentional?

File: templates/_components/static/list-events.twig [hunk 2]
```diff
@@ -17,11 +18,12 @@
...
             ]
         %}
-    
+
     {% for entry in content %}
         <li class="list-event">
             <div class="list-event__container | flex-grid">
-                <a href="#" class="list-event__media">
+                <a
+                    class="list-event__media" href="#">
```
Comment (johnnymueller) [inline templates/_components/static/list-events.twig]: Are line breaks here and above <ul> intentional?

File: templates/static/product-detail.twig [hunk 3]
```diff
@@ -83,7 +83,8 @@
         </article>
 
         {# Aside #}
-        <div class="entry-aside">
+        <div
+            class="entry-aside">
```
Comment (johnnymueller) [inline templates/static/product-detail.twig]: Line break and spacing above?

File: templates/_components/static/list-events.twig (hunk 1 above)
Comment (lindseybradford) [reply templates/_components/static/list-events.twig]: Fixed!

File: templates/_components/static/list-events.twig (hunk 2 above)
Comment (lindseybradford) [reply templates/_components/static/list-events.twig]: Fixed!

File: templates/static/product-detail.twig (hunk 3 above)
Comment (lindseybradford) [reply templates/static/product-detail.twig]: Fixed

**Pull request merged**